spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'User posts index page', type: :feature do
+  include ActionView::Helpers::TextHelper
+
+  let(:user) { User.create(name: 'John Doe', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Lorem ipsum', posts_counter: 3) }
+
+  before do
+    3.times do |i|
+      post = Post.create(author: user, title: "Test Post #{i}", text: 'This is a test post.')
+      3.times { Comment.create(user:, post:, text: 'This is a test comment.') }
+    end
+    visit user_posts_path(user)
+  end
+
+  scenario 'User sees user details on index page' do
+    expect(page).to have_css("img[src*='#{user.photo}']")
+    expect(page).to have_content(user.name)
+    expect(page).to have_content("Number of posts: #{user.posts_counter}")
+  end
+
+  scenario 'User sees post details on index page' do
+    expect(page).to have_content(Post.first.title)
+    expect(page).to have_content(Post.first.text)
+    expect(page).to have_content(Post.first.comments_counter)
+    expect(page).to have_content(Post.first.likes_counter)
+    expect(page).to have_content(Comment.first.text)
+  end
```

Comment:
- [ ] As the previous reviewer asked, this test should ensure you can see the first comments (more than one comment) for a user post, not only see the first comment as you did. Please fix that.